tests/test_simultaneous_initiate.c

```diff
@@ -122,6 +122,17 @@ START_TEST(test_basic_simultaneous_initiate)
     result = session_builder_process_pre_key_bundle(bob_session_builder, alice_pre_key_bundle);
```

Comment:
You missed adding the check to the other `session_builder_process_pre_key_bundle` function above this one